src/main/scala-2.12/org/ergoplatform/appkit/ergotool/dex/CancelOrderCmd.scala

```diff
@@ -170,7 +183,7 @@ object CancelOrder {
     if (outboxValue >= MinFee) {
```

Comment:
the name `unspentBoxes` in not readable.

Reply:
Renamed to `unspentBoxesForAmount`.